src/pages/cart/cart.js

```diff
@@ -9,6 +9,7 @@ export function CartPage() {
             <div className="container">
                 <Navbar />
             </div>
+            <h2>My Cart</h2>
             {cart.length > 0 ? TableBar() : EmptyBar()}
```

Comment:
i think instead of invoking as function here, you might write it as `<Component />`